### PR TITLE
cudf and cudf_kafka 0.20 now request 0.20 dependencies, and report as 0.20

### PR DESCRIPTION
**Pull request by robertmaynard**
Both cudf and cudf_kafka had a couple of issues when moving from version 0.19 to 0.20:

- cudf was still reporting as 0.19
- cudf was requesting rmm-0.19 for some conda packages
- cudf_kafka was requesting cudf version 0.19

### CODE REVIEW

**Review by jlowe — APPROVED** [review]
Java approval

File: cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetCUDF.cmake [hunk 1]
```diff
@@ -40,7 +40,16 @@ function(find_and_configure_cudf VERSION)
                         "BUILD_BENCHMARKS OFF")
     cudfkafka_restore_if_enabled(BUILD_TESTS)
     cudfkafka_restore_if_enabled(BUILD_BENCHMARKS)
+    if(cudf_ADDED)
+        set(cudf_ADDED TRUE PARENT_SCOPE)
+    endif()
 endfunction()
 
-set(CUDF_KAFKA_MIN_VERSION_cudf 0.19)
+set(CUDF_KAFKA_MIN_VERSION_cudf 0.20)
```
Comment (trxcllnt) [inline cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetCUDF.cmake:48]: What about this?
```suggestion
set(CUDF_KAFKA_MIN_VERSION_cudf "${CUDF_KAFKA_VERSION_MAJOR}.${CUDF_KAFKA_VERSION_MINOR}")
```

**Review by trxcllnt — CHANGES_REQUESTED** [review]
We should add these versions to the [`update-version.sh`](https://github.com/rapidsai/cudf/blob/branch-0.19/ci/release/update-version.sh) script.

**Comment by robertmaynard** [comment]
> We should add these versions to the [`update-version.sh`](https://github.com/rapidsai/cudf/blob/branch-0.19/ci/release/update-version.sh) script.

I will update the script, and use it to make the necessary changes.

File: cpp/CMakeLists.txt [hunk 2]
```diff
@@ -28,7 +28,7 @@ elseif(CMAKE_CUDA_ARCHITECTURES STREQUAL "")
   set(CUDF_BUILD_FOR_DETECTED_ARCHS TRUE)
 endif()
 
-project(CUDF VERSION 0.19.0 LANGUAGES C CXX)
+project(CUDF VERSION 0.20.0 LANGUAGES C CXX)
```
Comment (jdye64) [inline cpp/CMakeLists.txt:31]: This one confuses me? Seems this should have been updated as part of the release as per this release script https://github.com/rapidsai/cudf/blob/branch-0.19/ci/release/update-version.sh#L53

File: cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetCUDF.cmake (hunk 1 above)
Comment (jdye64) [reply cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetCUDF.cmake:48]: Should this also be updated as part of the `ci/release/update-version.sh` file?

Comment (robertmaynard) [reply cpp/libcudf_kafka/cmake/thirdparty/CUDF_KAFKA_GetCUDF.cmake:48]: I think the suggested change is the proper approach as that reduces the amount of places `ci/release/update-version.sh` needs to update.

File: cpp/CMakeLists.txt (hunk 2 above)
Comment (robertmaynard) [reply cpp/CMakeLists.txt:31]: Updated `update-version.sh` with the new name

**Review by raydouglass — APPROVED** [review]

**Review by trxcllnt — APPROVED** [review]

**Review by kkraus14 — APPROVED** [review]

**Comment by kkraus14** [comment]
@gpucibot merge

**Pull request merged**